home/.oh-my-zsh-custom/continuity.zsh-theme

```diff
@@ -7,8 +7,13 @@ if hash rvm-prompt 2>/dev/null; then
   local rvm_ruby='%{$fg[red]%}‹$(rvm-prompt s i v p g)›%{$reset_color%}'
 fi
 local git_branch='$(git_prompt_info)%{$reset_color%}'
+if type nvm >/dev/null 2>&1; then
+  local nvm_node='%{$fg[green]%}‹node-$(nvm current 2>/dev/null)›%{$reset_color%}'
+else
+  local nvm_node='%{$fg[green]%}‹node-$(node --version)›%{$reset_color%}'
```

Comment:
good point @benjaminoakes . `node_version_prompt` might be a bit more intent revealing